server/auth/register.js

```diff
@@ -22,7 +22,7 @@ const register = async ({ division, email, name, ctftimeId }) => {
       return responses.badKnownEmail
     }
     if (e.constraint === 'users_name_key') {
-      return responses.badKnownName
+      return [responses.badKnownName, name]
```

Comment:
In future, we could return the team id and name from the `/integrations/ctftime/callback` endpoint so that other parts of the platform which use CTFtime auth can access the data in the CTFtime token.